crates/aiken-project/src/error.rs

```diff
@@ -160,6 +162,28 @@ pub enum Error {
 
     #[error("I located conditional modules under 'env', but no default one!")]
     NoDefaultEnvironment,
+
+    #[error(
+        "I couldn't find the script corresponding to hash {script_hash} in your blueprint (plutus.json)."
+    )]
+    ScriptOverrideNotFound { script_hash: ScriptHash },
+
+    #[error("I couldn't parse the script override argument at index {index}...")]
+    ScriptOverrideArgumentParseError {
+        index: String,
+        #[source]
+        error: ScriptOverrideArugmentError,
+    },
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum ScriptOverrideArugmentError {
```

Comment:
I wasn't sure where this error type should live, so please tell me if there is a better place to leave it

Reply:
The place is good but we gotta fix the name.